functions/public/Invoke-WPFButton.ps1

```diff
@@ -50,13 +50,15 @@ function Invoke-WPFButton {
         "WPFFixesNetwork" {Invoke-WPFFixesNetwork}
         "WPFUpdatesdisable" {Invoke-WPFUpdatesdisable}
         "WPFUpdatessecurity" {Invoke-WPFUpdatessecurity}
+        "WPFWinUtilShortcut" {Invoke-WPFShortcut -ShortcutToAdd "WinUtil" -RunAsAdmin $true}
```

Comment:
This feature (Creating a Shortcut) have been removed in commit c186642998a77bcb95e59c46dd272f100ab50423, [link to exact line](https://github.com/ChrisTitusTech/winutil/commit/c186642998a77bcb95e59c46dd272f100ab50423#diff-70bdd2b079dc5748db02b2676484003945a7d561d8d903b3c742dadd69d30b24L53).

Reply:
I should've done that anyway... I'm not sure how I missed any of this. Would it be best to make a new PR or should I just push to this one?

Reply:
I recommend pushing to this branch and keeping this PR open, as it's currently a draft/work-in-progress PR (as stated by you). Also, things like this happen from time to time, so no worries 😄

Reply:
I know... But I still think a new PR is the best option, plus I can't wrap my head around conflict resolution.